### PR TITLE
Added a next prop named alternate text to receive alternate text for …

### PR DESCRIPTION
**Pull request by Gauravvpnd**
…button

<!---
Thank you for your pull request! 🚀

If you're not already a member of our public Matrix community, please consider joining via the following link:
https://matrix.to/#/#activist_community:matrix.org

It'd be great to have you!

### CODE REVIEW

**Review by andrewtavis — CHANGES_REQUESTED** [review]
Hey @Gauravvpnd! I'm realizing I didn't explain this as well as I should have. Specifically we're not looking for an alternate text for the button, but rather an alternate text that would be read by a screen reader for someone who's visibly impaired. I'll go into the code and explicitly describe what should happen :)

File: frontend/components/LabeledBtn.vue [hunk 1]
```diff
@@ -47,6 +48,10 @@ defineProps<{
   label: string;
   fontSize: string;
   linkTo: string;
+  alternateText: string; // New prop for alternate text
 }>();
 const localePath = useLocalePath();
+
+// Computed property for button text
+const buttonText = computed(() => alternateText || $t(label));
```
Comment (andrewtavis) [inline frontend/components/LabeledBtn.vue:56]: We don't need this line as the button's text should be `t$(label)` :)

File: frontend/components/LabeledBtn.vue [hunk 2]
```diff
@@ -47,6 +48,10 @@ defineProps<{
   label: string;
   fontSize: string;
   linkTo: string;
+  alternateText: string; // New prop for alternate text
```
Comment (andrewtavis) [inline frontend/components/LabeledBtn.vue:51]: This is great, but we don't need the comment saying it's "new".

File: frontend/components/LabeledBtn.vue [hunk 3]
```diff
@@ -37,7 +38,7 @@
       'text-base sm:text-lg xl:text-3xl xl:px-6 xl:py-3': fontSize == '3xl',
     }"
   >
-    {{ $t(label) }}
+    {{ buttonText }}
```
Comment (andrewtavis) [inline frontend/components/LabeledBtn.vue:41]: This should be reverted back to `$t(label)`.

File: frontend/components/LabeledBtn.vue [hunk 4]
```diff
@@ -17,7 +18,7 @@
...
```
Comment (andrewtavis) [inline frontend/components/LabeledBtn.vue:21]: This should be reverted back to `$t(label)`.

File: frontend/components/LabeledBtn.vue [hunk 5]
```diff
@@ -1,3 +1,4 @@
+
 <template>
   <a
```
Comment (andrewtavis) [inline frontend/components/LabeledBtn.vue:3]: This `<a>` tag should be given an `aria-label` attribute that has `alternateText` assigned to it 😊

File: frontend/components/LabeledBtn.vue [hunk 6]
```diff
@@ -17,7 +18,7 @@
       'text-base sm:text-lg xl:text-3xl xl:px-6 xl:py-3': fontSize == '3xl',
     }"
   >
-    {{ $t(label) }}
+    {{ buttonText }}
   </a>
   <NuxtLink
```
Comment (andrewtavis) [inline frontend/components/LabeledBtn.vue:23]: This `<NuxtLink>` tag should be given an `aria-label` attribute that has `alternateText` assigned to it 😊

**Comment by andrewtavis** [comment]
A short article from the World Wide Web Consortium can be seen here: https://www.w3.org/WAI/WCAG22/Techniques/aria/ARIA8

Let me know on the above! As soon as we have the above changes I'd be happy to merge, and then from there you can let me know if you have interest in writing alternate texts for some of the `LabeledBtn` components in the code base :)

**Comment by andrewtavis** [comment]
Hey @Gauravvpnd 👋 Checking in with you to see how you're doing with all this. Very happy to assist if there's something I can do to help. Just let me know :)

**Comment by andrewtavis** [comment]
@Gauravvpnd, checking in again to see how you're doing :) If you're unsure about the changes we could also do a quick call or I could explain things in more detail 😊 Hope you're well!

**Comment by Gauravvpnd** [comment]
> changes

Hey,I'm not able to get what are the specific changes need to do it will be helpful if you will tell me a little on what to do. I am not able to get your point.

**Comment by andrewtavis** [comment]
Something else that I'm noticing is that you don't seem to have your email for GitHub set up properly. The [email that you use for GitHub](https://github.com/settings/emails) should be the same that you have for `git config --global user.email` or locally for the repo :)

**Comment by andrewtavis** [comment]
I’d again be happy to discuss all this over a call, @Gauravvpnd :) Just let me know what your time zone is and we can figure out a time that would work for both of us.

**Comment by andrewtavis** [comment]
Hi @Gauravvpnd 👋 I’m going to close this PR now as there are conflicts that are unresolved. There’s some interest from people to maybe work on this, and I might need to assign them. Would be happy to work with you on another issue in the future! Please be in touch and thanks for your interest in activist :)

**Pull request closed without merging**